utils/scripts/load-binary.sh

```diff
@@ -216,6 +216,12 @@ elif [ "$TARGET" = "golang" ]; then
 
     echo "Using github.com/DataDog/dd-trace-go/v2@$COMMIT_ID"
     echo "github.com/DataDog/dd-trace-go/v2@$COMMIT_ID" > golang-load-from-go-get
+    echo "github.com/DataDog/dd-trace-go/v2/contrib/IBM/sarama.v1@$COMMIT_ID" > golang-load-from-go-get
+    echo "github.com/DataDog/dd-trace-go/v2/contrib/gin-gonic/gin@$COMMIT_ID" > golang-load-from-go-get
+    echo "github.com/DataDog/dd-trace-go/v2/contrib/go-chi/chi.v5@$COMMIT_ID" > golang-load-from-go-get
+    echo "github.com/DataDog/dd-trace-go/v2/contrib/google.golang.org/grpc@$COMMIT_ID" > golang-load-from-go-get
+    echo "github.com/DataDog/dd-trace-go/v2/contrib/labstack/echo.v4@$COMMIT_ID" > golang-load-from-go-get
+    echo "github.com/DataDog/dd-trace-go/v2/contrib/net/http@$COMMIT_ID" > golang-load-from-go-get
```

Comment:
@knusbaum I think we should use `>>` instead of `>` to avoid overwriting the file instead of adding lines.

Reply:
Doh! You're exactly right.